appengine/hello-world/standard/package.json

```diff
@@ -15,15 +15,17 @@
   "scripts": {
     "deploy": "gcloud app deploy",
     "start": "node app.js",
-    "system-test": "repo-tools test app",
+    "system-test": "mocha --exit test/*.test.js",
```

Comment:
The `--exit` always makes me nervous, but it's not really blocking

Reply:
These tests don't finish without `--exit`

Reply:
The default for `--exit` was changed relatively recently in mocha:

https://github.com/mochajs/mocha/pull/3034

I think it's reasonable to go back to the old default ... alternatively you'd need to capture the `server` object returned by `app.listen` and then call `.close()` on it when tests are complete... sounds like a hassle :stuck_out_tongue:

Reply:
So by using `--exit` I'm just using Mocha the way it used to be, right?